### ISSUE TITLE
A11y_Visual Studio Code Services_Home_Keyboard: The "Get Started" control is a link but visually it appears as a button.

### ISSUE THREAD

**Issue by Yashi1201** [microsoft/vscode-docs#5555]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win10;#FTP;#DesktopWeb;#A11ySev2;#WCAG3.2.4;#Keyboard;#Chrome;

## Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Windows Version: Win10
Chrome Version: Version 103.0.5060.134(Official build)(64-bit)

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Launch the application Url : https://code.visualstudio.com/
3. Tab till "Get Started" link.
4. Observe that "Get Started" is a link but visually appearing as button.

## Actual:
The "Get Started" control is a link but visually appears as a button. The visual appearance of this control as a button is not consistent.
### Similar issue is observed in following scenario:

1.  Home_Header: The "Download" control is a link but visually it appears as a button.
2. Docs Python_Tutorial: The "Edit" control  is a link but visually it appears as a button.

## Expected:
The "Get Started" control is a link and visually should also appear as a link. All links should be consistent in appearance visually.

## User Impact:
Keyboard users will get confused with the control appearance as a button and function as a link.

## Attachment
![Get started link appearing as button visually](https://user-images.githubusercontent.com/93374043/183345989-698d376f-25f4-471f-a212-6b50551ca7ae.gif)
![Similar issue Download is a link but visually appearing as button](https://user-images.githubusercontent.com/93374043/183346476-247ae090-b2ce-4a5d-9399-ab24f9f00f51.gif)
![5555_A11y_Home_Keyboard_Similar issue_Docs-Python](https://user-images.githubusercontent.com/93374043/183351486-ea9bb5e5-3d34-47ee-945b-5bd4c7e7e76b.gif)

**Comment by gregvanl** [comment]
Won't fix. Many sites including learn.microsoft.com (see image) below use buttons for links.
Closing as GitHub issue do not have a "Resolved" state.
Here the "Edit this document" button on learn.microsoft.com is a link that takes the user to the Markdown file on GitHub
![image](https://user-images.githubusercontent.com/12818376/191765627-18e9579b-0edf-4d3d-ada0-590f7f8b120c.png)

**Comment by Yashi1201** [comment]
As per discussion with SME, "If there is a link and it is visually appearing as button control then it is NAB as long as screen reader should read the correct role for it, visually they can appear as anything".
